testflinger_cli/__init__.py

```diff
@@ -274,6 +274,7 @@ def cancel(self, job_id=None):
                     "Received 404 error from server. Are you "
                     "sure this is a testflinger server?"
                 ) from exc
```

Comment:
just in case, you can use `sys.exit` with a message as an argument (https://docs.python.org/3.4/library/sys.html#sys.exit)

Reply:
It's totally fine to do it in another PR!